### DISCUSSION TITLE
**🚀 Issue Bounty Program! 🏆**

### DISCUSSION THREAD

**Discussion by myanzik** [rahataid/rahat-platform#192] [Announcements]
We’re thrilled to announce a bounty program for Rahat! We’re inviting all contributors to create **detailed issues** in our repository, whether they are **bugs** or **Rahat Improvement Proposals (RIPs)**. 

You can **earn bounties** for each well-documented issue you submit during the hackathon!

**How to participate:**

1. **Create an issue**:
   - **Bug**: Found a bug or unexpected behavior? Submit a bug issue with a clear description of the problem, steps to reproduce, expected behavior, and actual results.
   - **Rahat Improvement Proposal (RIP)**: Got an idea for improving Rahat? Propose a new feature or improvement and explain why it will add value to the platform.

2. **Use the templates**: Follow the **Bug** or **RIP** templates provided in the repository to ensure we capture all the details needed to address your issue.

3. **Earn your bounty**: After your issue is reviewed and assigned to someone, you’ll earn a bounty as a token of appreciation for your contribution!

---

**What makes an issue eligible for a bounty?**

- **Clear and detailed descriptions** with all necessary information.
- **Steps to reproduce** for bug reports.
- **Clear reasoning and impact analysis** for improvement proposals.
- **Proposed solutions or implementations** are a big plus for improvement issues!

**Example of a good issue description:**
- **Title**: “Bug: Incorrect token balance after transaction”
- **Description**:
   - **Steps to reproduce**: 
     1. Navigate to the dashboard.
     2. Send 10 tokens to another user.
     3. Refresh the page.
   - **Expected result**: Token balance should be updated immediately.
   - **Actual result**: Token balance is not updating after the transaction.
   - **Environment**: Rahat vX.Y.Z (specify version if possible).

---

**Let’s work together to make Rahat even better!** Your contributions are crucial to the success of this project, and we can’t wait to see your amazing ideas and solutions.

Join the fun, submit your issues, and let’s build something impactful!

Good luck, and happy hacking! 🎉👩‍💻👨‍💻

**Comment by gursuj** [comment]
Pretty cool. For which repos is the bounty available? Only [rahat-platform](https://github.com/rahataid/rahat-platform)?

**Reply by manjik-rumsan** [reply]
[rahat-platform](https://github.com/rahataid/rahat-platform) and [rahat-ui](https://github.com/rahataid/rahat-ui)

**Comment by redbeans9** [comment]
hlo any bounty for solidity issues  
#113 #112 #111
